idaes/surrogate/__init__.py

```diff
@@ -1,22 +0,0 @@
-#################################################################################
```

Comment:
This is probably due to merging the recent reorganization changes into this PR and might be OK, but I am concerned to see these changes showing up in this PR.

Reply:
To be safe should I could make a new branch off main, copy the new pruning/sparsification files, and submit a new PR? Just to be sure the only changes are the new files.